client/src/Components/Header.js

```diff
@@ -0,0 +1,83 @@
+import React from "react";
+import { makeStyles } from "@material-ui/core/styles";
+import AppBar from "@material-ui/core/AppBar";
+import Toolbar from "@material-ui/core/Toolbar";
+import IconButton from "@material-ui/core/IconButton";
+import Typography from "@material-ui/core/Typography";
+import SearchIcon from "@material-ui/icons/Search";
+import SettingsIcon from "@material-ui/icons/Settings";
+import TextField from "@material-ui/core/Input";
+import Grid from "@material-ui/core/Grid";
+
+const useStyles = makeStyles((theme) => ({
+  typography: {
+    fontFamily: theme.typography.fontFamily,
+  },
+  palette: {
+    background: theme.palette.primary.main,
+  },
+}));
+
+export default function Header() {
+  const classes = useStyles();
+  return (
+    <div>
+      <AppBar
+        className={classes.palette}
+        position="static"
+        style={{ height: "4em" }}
+      >
+        <Toolbar>
+          <Grid container spacing={0}>
+            <Grid item xs={4} style={{ display: "flex", alignItems: "center" }}>
+              <Typography className={classes.typography} variant="h6">
+                <span style={{ color: "white" }}>mentions</span>
+                <span style={{ color: "#191970" }}>crawler.</span>
```

Comment:
Since we have a style object, I would just use className everywhere in this component, rather than inline.